app/model/Travel/Command/TravelDetails.php

```diff
@@ -29,9 +30,9 @@ class TravelDetails
     private $date;
 
     /**
-     * @ORM\Column(type="string", name="type", length=5)
-     *
-     * @var string
+     * @ManyToOne(targetEntity="Type")
```

Comment:
`ManyToOne` určitě není správná vazba. Mělo by to být `ManyToMany`.

Jinak obecně bych se zamyslel, jestli nejsme schopni ten typ reimplementovat jako

Reply:
Proč by nefungovala many to one kdyz je to jednosmerne? reimplementovat to muzeme, ale nyní jsem to chtel hlavně fixnout. Tak muzu upravit, aby to vracelo jen opravdu stringy, tak se předpokládalo